lib/Interpreter/DeclUnloader.cpp

```diff
@@ -649,7 +649,12 @@ bool DeclUnloader::VisitRedeclarable(clang::Redeclarable<T>* R, DeclContext* DC)
     GD = GlobalDecl(CXXDtor, Dtor_Base);
     MaybeRemoveDeclFromModule(GD);
     GD = GlobalDecl(CXXDtor, Dtor_Comdat);
+#if defined(LLVM_ON_WIN32)
+    // MicrosoftMangle.cpp:954 calls llvm_unreachable when mangling Dtor_Comdat
```

Comment:
We should teach `utils::Analyze::maybeMangleDeclName` about this current (?) limitation of Windows mangler and return the non-mangled name if the dtor is Comdat and we are using the Windows mangler. This way other users of `utils::Analyze::maybeMangleDeclName` would benefit from this.